ironfish/src/mining/pool.ts

```diff
@@ -438,19 +438,6 @@ export class MiningPool {
       ),
     )
 
-    // Target might be the same if there is a slight timing issue or if the block is at max target.
-    // In this case, it is detrimental to send out new work as it will needlessly reset miner's search
-    // space, resulting in duplicated work.
```

Comment:
This comment was incorrect, since by changing the timestamp below, we are changing the preimage of the hash function, changing the search space.